src/background/providers/cloudflare.ts

```diff
@@ -242,21 +243,21 @@ export class CloudflareProvider implements Provider {
             // header according to https://xhr.spec.whatwg.org/#dom-xmlhttprequest-setrequestheader
             // So we need to extract the token from the Referer header and send it in the query
             // param __cf_chl_f_tk instead. (Note that this token is not a Privacy Pass token.
-            let token: string | null = null;
+            let atoken: string | null = null;
```

Comment:
What does this mean? Is it a lint error? I cannot find it from `npm run lint`?

Reply:
I think `token` (without s) already means a token.

Reply:
Linter reports `security/detect-possible-timing-attacks` when detects a variable named `token` or `password`. 

https://github.com/gkouziik/eslint-plugin-security-node/blob/master/lib/rules/detect-possible-timing-attacks.js